src/h5web/vis-packs/core/shared/TooltipMesh.tsx

```diff
@@ -78,7 +78,7 @@ function TooltipMesh(props: Props) {
         <meshBasicMaterial opacity={0} transparent />
         <planeGeometry args={[width, height]} />
       </mesh>
-      <Html>
+      <Html style={{ width, height }}>
```

Comment:
I don't believe you actually need this, because the `svg` of the guides has a `width` and `height`, which makes the inner `div` of `Html` fullscreen.